Tests/ScipioKitTests/DebugSymbolTests.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 @testable import ScipioKit
 import Testing
 
+@Suite(.serialized)
 struct DwarfExtractorTests {
```

Comment:
Why these `.serialized` traits are added / required?

Reply:
swift-testing runs tests in parallel in default. As I investigate, it may cause CI hung. So it's safe to annotate `serialized` for all test cases to execute the build system. It may not be thread-safe.